scripts/qa_crowbarsetup.sh

```diff
@@ -3666,7 +3666,12 @@ function oncontroller_magnum_service_setup ()
         # os-distro property
         openstack image create --file $service_image_name \
             --disk-format qcow2 --container-format bare --public \
-            --property os_distro=fedora-atomic magnum-service-image
+            --property hw_firmware_type=uefi \
+            --property os_distro=jeos magnum-service-image
+    fi
+
+    if ! nova keypair-show controller-ssh; then
```

Comment:
We already create a keypair which is used by the magnum tempest tests. See https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L3687